examples/compose.js

```diff
@@ -0,0 +1,53 @@
+import merge from 'lodash/object/merge';
+import map from 'lodash/collection/map';
+
+const getDescriptorProps = (descriptorName, composables) => {
+  return map(composables, 'compose.' + descriptorName);
+};
+
+const createStamp = descriptor => {
+  return ({ instance } = {}) => {
+    const proto = {};
+    const obj = instance || {};
+
+    // Set up prototype
+    Object.assign(proto, descriptor.methods);
+    Object.setPrototypeOf(obj, proto);
```

Comment:
You'd need to check if the object already have a prototype. It should not be overriden.
```js
if (Object.getPrototypeOf(obj) === Object.prototype)
```